src/fshtypes/rules/FixedValueRule.ts

```diff
@@ -1,7 +1,7 @@
 import { Rule } from './Rule';
-import { Code, Quantity, Ratio } from '../index';
+import { Code, FshQuantity, FshRatio } from '../index';
```

Comment:
I understand why we wanted to add the `Fsh` prefix to these classes, so the names don't conflict with the FHIR types. That said, is there a reason we don't also do that with `Code`? Is there no FHIR type for `Code`? I wonder if we should name it `FshCode` just for consistency unless there's a reason not to do that.

Reply:
I wouldn't be opposed to adding `Fsh` prefixes to other FSH types that might conflict.  The reason `Code` doesn't generally conflict is that in FHIR the `code` type is really just a string -- so we didn't really need to build a type around it.